packages/terra-graphs-docs/CHANGELOG.md

```diff
@@ -20,3 +20,4 @@
 
 * Changed 
   * Moved core documentation to terra-dev-site.
+  * Changed documentation for General graph to be carbon-centric.
```

Comment:
nit: `Updated general graph documentation and examples to be carbon-centric.`

Reply:
Fixed [here](https://github.com/cerner/terra-graphs/pull/184/files#diff-dc22343a9102f2186f374011efbd61a1f3ac9df233e1e9d9385065fd4e2284d1R23).